package.json

```diff
@@ -9,7 +9,6 @@
   "devDependencies": {
     "coveralls": "^2.11.15",
     "http-ndjson": "^3.0.0",
-    "pre-commit": "^1.1.2",
```

Comment:
"it just crashes on linux" - @Rauno56 can you elaborate? Do you mean it didn't allow you to commit .. that may be because of the linting issue...